### DISCUSSION TITLE
How to make wire:target and @entangle work together?

### DISCUSSION THREAD

**Discussion by lalov** [livewire/livewire#4236] [1. Help]
I have a component where a property is entangled. When using wire:loading on a DIV, the DIV displays when the property is changed (and Livewire posts the changes). 
If I also use wire:target='nameofproperty', then wire:loading doesn't work
Is there a way to target the wire:loading when using @entangle ?

**Comment by joshhanley** [comment]
@lalov I'm pretty sure `wire:target` won't work with `@entangled` properties.

**Reply by lalov** [reply]
Yes, thanks for answering! Is it possible to emit some javascript/alpine event to mimic the behaviour when wire:target is used with standard wire:model? As a workaround?

**Reply by joshhanley** [reply]
@lalov yep probably, but I've not done it. Have a look at the javascript lifecycle hooks here, you might be able to use them to manually do your own loading https://laravel-livewire.com/docs/2.x/lifecycle-hooks
